params.go

```diff
@@ -103,7 +103,7 @@ func setParameter(msg protoreflect.Message, fields []protoreflect.FieldDescripto
 }
 
 func unmarshalFieldValue(msg protoreflect.Message, field protoreflect.FieldDescriptor, data []byte) (protoreflect.Value, error) {
-	switch kind := field.Kind(); kind {
+	switch kind := field.Kind(); kind { //nolint:exhaustive // All supported types are covered.
```

Comment:
I think a better way to address this is to actually add the last kind below.
```diff
-	case protoreflect.MessageKind:
+	case protoreflect.MessageKind, protoreflect.GroupKind:
```
The reason you should do this is in case someone defines an Editions file that uses group encoding with a well-known type:
```proto
edition = "2023";
import "google/protobuf/wrappers.proto";
message Foo {
  google.protobuf.Int64Value i = 1 [features.message_encoding=DELIMITED];
}
```